spec/javascript/components/supplemental_application/__snapshots__/SupplementalApplicationPage.test.js.snap

```diff
@@ -2382,233 +3542,364 @@ exports[`SupplementalApplicationPage Status Modal should render the status updat
                                             "state": "CA",
-                                            "street": "123 MAIN ST",
-                                            "zip_code": "94105-1804",
-                                          },
-                                          Object {
-                                            "agency_name": undefined,
-                                            "alternate_contact_type": undefined,
-                                            "alternate_contact_type_other": undefined,
-                                            "city": "buenos aires",
-                                            "date_of_birth": Object {
-                                              "day": "11",
-                                              "month": "06",
-                                              "year": "1976",
-                                            },
-                                            "email": undefined,
-                                            "first_name": "diego",
-                                            "id": "a0n0x000000XzchAAC",
-                                            "last_name": "maradona",
-                                            "mailing_address": undefined,
-                                            "mailing_city": undefined,
-                                            "mailing_state": undefined,
-                                            "mailing_street": undefined,
-                                            "mailing_zip_code": undefined,
-                                            "marital_status": undefined,
-                                            "middle_name": "m",
-                                            "name": "diego m maradona",
-                                            "phone": undefined,
-                                            "phone_type": undefined,
-                                            "primary_language": undefined,
-                                            "relationship_to_applicant": "Child",
-                                            "residence_address": undefined,
-                                            "second_phone": undefined,
-                                            "second_phone_type": undefined,
-                                            "state": "fl",
-                                            "street": "9 de julio",
-                                            "zip_code": "33026",
+                                            "street": "123 Test St",
+                                            "zip_code": "00000",
                                           },
                                         ],
                                         "assignSupplementalAppTouched": [Function],
-                                        "availableUnits": Array [],
+                                        "availableUnits": undefined,
```

Comment:
availbale units being undefined here worries me a bit

Reply:
this appears to be fine on the site, not sure what's going on with the snapshot but I'm OK with it for now